build.py

```diff
@@ -1,19 +1,19 @@
 #!/usr/bin/env python
-
+import itertools
 import json
 from subprocess import run
 from variables import php_releases
 
 for php_release in php_releases:
-    command = ["docker", "pull", php_release]
+    command = ["docker", "pull", f"php:{php_release}"]
+    run(command)
 
 image = "michaelmcandrew/civicrm"
 combos = json.load(open("combos.json"))
 
 for key, combo in combos.items():
-    # tags = {x for x in combo["tags"]}
     command = ["docker", "build", combo["dir"]]
-    for tag in combo["tags"]:
-        command.extend(["-t", f"{image}:{tag}"])
-        run(command)
-
+    command += list(
```

Comment:
any particular reason for this change?

Reply:
Probably I was playing with it while making changes and just left it like this without realising I had changed it in the past.